src/main/java/webs/hillel/Student.java

```diff
@@ -0,0 +1,46 @@
+package webs.hillel;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "student")
+public class Student {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
```

Comment:
I would suggest adding not null restriction as it doesn't make any sense to create a student without a name